servo/connectors/kubernetes.py

```diff
@@ -2710,7 +2769,7 @@ async def apply(self, adjustments: List[servo.Adjustment]) -> None:
                         *list(map(lambda a: a.apply(), self.optimizations)),
                         return_exceptions=True,
                     ),
-                    timeout=timeout.total_seconds(),
+                    timeout=timeout.total_seconds() + 60, # allow sub-optimization timeouts to expire first
```

Comment:
Prolly want to put the 60 into a constant since its referenced multiple times as a literal

Reply:
Agree